src/apps/backend/modules/otp/internal/store/otp-repository.ts

```diff
@@ -0,0 +1,10 @@
+import { ApplicationRepository } from '../../../application';
+
+import { OtpDB, OtpDbSchema } from './otp-db';
+
+const OtpRepository = ApplicationRepository<OtpDB>(
+  'Otp',
```

Comment:
We have been using plural name for collections, all lower case, this should be `otps`